README.md

```diff
@@ -1,63 +1,280 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
-validate contact form
-> Description the project.
-Adding basic javaScript for form validation
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-## Built With
-Basic HTML & CSS & JavaScript
-- Major languages
-- Frameworks
-- Technologies used
-VSCode
-## Live Demo (if available)
+Modify this file to match your project and remove sections that don't apply.
 
-## Getting Started
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-To get a local copy up and running follow these simple example steps.
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project name] <a name="Building-a-Portfolio"></a>
```

Comment:
- [ ] As the previous reviewer mentioned, Please edit the project name (because it's not appearing right), built with, getting started, and other sections in your readme file and remove the unused sections from it to make it more professional.

![image](https://user-images.githubusercontent.com/68796232/204111180-7df5b9f8-9d0e-4b9d-aa4b-7d201f4d95a3.png)